### PR TITLE
chore: add release notes for 1.31.0

### PR DESCRIPTION
**Pull request by pahnin**
JIRA https://smartbear.atlassian.net/browse/PACT-2280

### CODE REVIEW

File: website/docs/docs/on-premises/releases/1.31.0.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+2. Documents the API which was missed previously ( /pacticipants/:pacticipant_name/labels/:label_name ) for managing labels for a pacticipant.
+* Fixed unique team name issue when updating team in settings
+* 1. Bumps openapi_first to major version 2.1
+null
+2. Fixes the tests that were failing when updating the openapi_version beyond 1.0
+* Fix bug which caused a missing row in the matrix when there were a mix of classic Pact and bi-directional contracts
+* Resolves the `UniqueConstraintViolation` issue when updating team users via a PUT request.
+* Fixed incorrect type standardization for openapi -v 3.1.0
+* Team Administrators will now have permission to view environments.
+* Resolves the authentication issue for newly invited users with sbid: true and no IDP subject.
```
Comment (Ilia) [inline website/docs/docs/on-premises/releases/1.31.0.md:26]: I think we can remove this, onprem doesn’t have sbid

File: website/docs/docs/on-premises/releases/1.31.0.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+* Resolution: Implement case-insensitive matching for existing SBID users experiencing login issues.
```
Comment (Ilia) [inline website/docs/docs/on-premises/releases/1.31.0.md:27]: Same here

File: website/docs/docs/on-premises/releases/1.31.0.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+22 August 2024
+
+## Features
+
+* added new /labels api that lists down all the labels
+
+## Fixes
+
+* 1. Fixes the NoMethodError for pacticipants PATCH request when request body contains embedded labels. 
+null
```
Comment (Ilia) [inline website/docs/docs/on-premises/releases/1.31.0.md:16]: Remove the nulls

File: website/docs/docs/on-premises/releases/1.31.0.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+22 August 2024
+
+## Features
+
+* Added new `/labels` API that lists all labels
+
+## Fixes
+
+* Fixes an issue where the "next" embedded link was not documented in the response body of the `GET /audit`
```
Comment (mefellows) [inline website/docs/docs/on-premises/releases/1.31.0.md:15]:
```suggestion
* Fixes an issue where the "next" embedded link was not documented in the response body of the `GET /audit` endpoint
```

File: website/docs/docs/on-premises/releases/1.31.0.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+* Fixes the NoMethodError for pacticipants PATCH request when request body contains embedded labels
```
Comment (mefellows) [inline website/docs/docs/on-premises/releases/1.31.0.md:16]:
```suggestion
* Fixes the `NoMethodError` for pacticipants PATCH request when request body contains embedded labels
```

File: website/docs/docs/on-premises/releases/1.31.0.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+* Documents the API which was missed previously ( /pacticipants/:pacticipant_name/labels/:label_name ) for managing labels for a pacticipant
```
Comment (mefellows) [inline website/docs/docs/on-premises/releases/1.31.0.md:17]:
```suggestion
* Documents the API `/pacticipants/:pacticipant_name/labels/:label_name` for managing labels for a pacticipant
```

**Review by mefellows — APPROVED** [review]
Made a few minor suggestions, but LGTM otherwise. Any migrations need to be documented also.

**Pull request merged**